### PR TITLE
Closes #37 vision and boundaries

### LINKED ISSUE #37: Add vision and boundaries
Add a vision for the project with references to the overall platform description in the introduction.

Include arguments on why this product is better than the existing.

### PR DESCRIPTION
**Pull request by Karnich**
Let me know what you think. Also i think we need to rewrite stuff like the introduction to get a red string?

### CODE REVIEW

File: files/vision.tex [hunk 1]
```diff
@@ -0,0 +1,23 @@
+\chapter{Vision}
+This chapter will give insights into the vision this project is based upon and the boundaries for what this project will focus upon.
+
+
+\section*{The big picture}
+
+The vision behind this project is to revolutionize the club management and progress monitoring inside badminton clubs. The main part where this product will be different is a smart racket that can give the club, coach and individual player a bunch of information about their badminton performance. This information can help the coaches finding weaknesses and show them where to focus their training on a specific player.
```
Comment (Crevil) [inline files/vision.tex]: different from what?
There are other rackets out there, are we any different?

File: files/vision.tex [hunk 2]
```diff
@@ -0,0 +1,23 @@
+\chapter{Vision}
+This chapter will give insights into the vision this project is based upon and the boundaries for what this project will focus upon.
+
+
+\section*{The big picture}
+
+The vision behind this project is to revolutionize the club management and progress monitoring inside badminton clubs. The main part where this product will be different is a smart racket that can give the club, coach and individual player a bunch of information about their badminton performance. This information can help the coaches finding weaknesses and show them where to focus their training on a specific player. 
+
+The management portion would be a web application where management staff could handle business such as economy, practice attendance and events such as parties and game scheduling.
```
Comment (Crevil) [inline files/vision.tex]: I think 'portion' should be 'part'
As this is the vision, I think we can phrase it as 'is' instead of 'would', because this is what we want to do, but as you limit the scope later on, it is still clear it will not be done in this project, ?

File: files/vision.tex [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The progress monitoring will be made possible with a mobile application that can gather data over bluetooth from sensors in the smart racket. The data will not only be available on the mobile device, it will also be available in the web application, but the mobile device is required to gather the data in the first place. The club can own smart rackets for their players to use on occasions during practice. The coach then gathers this data for both the player and club to see. The player should also have the option to buy his own smart racket so that he can monitor his progress at every single practice or match.
```
Comment (Crevil) [inline files/vision.tex]: Bluetooth in italics
Last phrase: 'to buy his' should be 'to buy its' or 'to buy his/hers'

File: files/vision.tex [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+Data will bind to the players individual user and will be shareable with the coaches and club. Players can compare their progress and stats with their friends and other players from the club. It should also be possible to share training sessions and progress on social media.
```
Comment (Crevil) [inline files/vision.tex]: "players" -> "player's"

File: files/vision.tex [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+\graphic{0.85}{vision.png}{Illustration of the projects vision}{vision}
```
Comment (Crevil) [inline files/vision.tex]: There is no reference to this image? Could it be inserted in the beginning?

File: files/vision.tex (hunk 1 above)
Comment (Karnich) [reply files/vision.tex]: well that is standalone rackets. Ours integrate with a management system :p

File: files/vision.tex [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+\section*{Boundary}
+Since the vision is more than 2 man can swallow during this project duration. Boundaries are set for the specific part of the vision that we want to achieve during this project.
```
Comment (Crevil) [inline files/vision.tex]: '2' -> 'two'
'swallow' is very non-formal. Maybe something like: "As the vision covers more than two men can overcome during the projects duration, boundaries(...)" ?
Maybe add a reference to the last vision illutstration with the limits here as well?

File: files/vision.tex [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+As mentioned in the vision, the part that makes this product different is the progress monitoring through the mobile application and smart racket. Since this is the unique feature, it is also gonna be our focus point for this project.
```
Comment (Crevil) [inline files/vision.tex]: different from what?
I think the last phrase is a bit non-formal. Gonna is "slang" for "going to". Maybe rephrased to something like: "it is the main focus of this project" ?

File: files/vision.tex [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+To get this product introduced into the clubs we have allied with external consultants that will visit clubs to show the product and introduce staff and players to how it works and what they can gain by using our product
```
Comment (Crevil) [inline files/vision.tex]: 'into' -> 'to'
'our' -> 'the'

File: files/vision.tex [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+\graphic{0.8}{vision_focus.png}{Illustration of the projects boundary}{vision}
```
Comment (Crevil) [inline files/vision.tex]: Not referenced, see earlier for a possible place to reference it.
I think the illustration in "Vision_with_Focus" is more clear than this one? It shows a clear relation to the overall vision.

**Comment by Crevil** [comment]
I like it!
There are two images of the limited scope of vision, but only one is used. A mistake or just to keep both of them?

File: files/vision.tex (hunk 1 above)
Comment (Crevil) [reply files/vision.tex]: Write that!
